frontend/src/views/MyAreaPage.vue

```diff
@@ -0,0 +1,207 @@
+        <div class="flex flex-wrap gap-8 p-8 justify-center mobile:hidden">
+
+
+            <AREAInfoComponent v-for="action in actions" :key="action.name" :values="action" :icon="action.icon"
+                :color="action.color" @menu-click="handleMenuClick(action.name)"
+                @more-click="handleMoreClick(action.name)" @configure-click="handleConfigureClick(action.name)" />
+        </div>
+    </div>
+</template>
+
+
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useRouter } from 'vue-router';
+import { Icon } from '@iconify/vue';
+
+import ServiceNavComponent from '@/components/ServiceNavComponent.vue';
+import MobileServiceNavComponent from '@/components/MobileServiceNavComponent.vue';
+import ServiceNavScrollComponent from '@/components/ServiceNavScrollComponent.vue';
+import RateComponent from '@/components/RateComponent.vue';
+import SaveComponent from '@/components/SaveComponent.vue';
+import ArrowComponent from '@/components/ArrowComponent.vue';
+import AREAInfoComponent from '@/components/AREAInfoComponent.vue';
+import AREACreationComponent from '@/components/AREACreationComponent.vue';
+
+const router = useRouter();
+
+const color = ref<string>('#1C1C53');
+const name = ref<string>('MyArea');
+const logo = ref<string>('mdi:home-modern');
+const rate = ref<number>(4.5);
+const reviews = ref<number>(120);
+const saves = ref<number>(50);
+const isActivated = ref<boolean>(true);
+const nbActions = ref<number>(5);
+
+const nameCapitalized = ref(name.value.toUpperCase());
+const isHeroVisible = ref(true);
+
+const isCircleFirst = ref(true);
+const scrollY = ref(0);
+
+window.addEventListener('scroll', () => {
+    scrollY.value = window.scrollY;
+})
+
+const handleClick = () => {
+    isActivated.value = !isActivated.value;
+    isCircleFirst.value = !isCircleFirst.value;
+}
+
+const openServicePage = () => {
+    console.log('Service page opened');
+    isHeroVisible.value = false;
+}
+
+const handleBackButtonSecondPage = () => {
+    console.log('Back button clicked on second page');
+    isHeroVisible.value = true;
+}
+
+function handleMenuClick(name: string) {
+    console.log('Menu clicked on:', name);
+}
+
+function handleMoreClick(name: string) {
+    console.log('More clicked on:', name);
+}
+
+function handleConfigureClick(name: string) {
+    console.log('Configure clicked on:', name);
+}
+
+function handleBackButtonFirstPage() {
+    console.log('Back button clicked on first page');
+    router.push('/dashboard');
+}
+
+function handleRedirectUserPage() {
+    console.log('Redirecting to user page');
+    router.push('/userinfo');
+}
+
+function handleScrollAttempt(event: WheelEvent) {
+    if (event.deltaY > 0) {
+        openServicePage();
+    }
+}
+
+function handleScrollAttemptSecondPage(event: WheelEvent) {
+    if (event.deltaY < 0 && scrollY.value === 0) {
+        isHeroVisible.value = true;
+    }
+}
+
+const actions = ref([
```

Comment:
It's areas and should be retrieved from the server (ask Adam about it)

On a route entitled : /user/areas/

Reply:
You wont have the color or the icon doing that, you must look inside the service store for those info.

Reply:
we wait adam